packages/conf-time/conf-time.1/opam

```diff
@@ -0,0 +1,13 @@
+opam-version: "1.2"
+maintainer: "Gabriel Scherer <gabriel.scherer@gmail.com>"
+authors: ["POSIX"]
+homepage: "https://github.com/ocaml/opam-repository/"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+dev-repo: "https://github.com/ocaml/opam-repository.git"
+license: "GPL"
+build: [["which" "time"]]
+depexts: [
+  [["debian"] ["time"]]
+  [["ubuntu"] ["time"]]
```

Comment:
It would be nice to extend the depexts a bit for other platforms distributions, [here's](https://gist.github.com/dbuenzli/eecc31ab3ee10ef3ccfe60efcbc51630) a list of system package manager online search that can help.